src/main/java/fr/insee/pearljam/infrastructure/campaign/jpa/CommunicationRequestJpaRepository.java

```diff
@@ -82,7 +86,8 @@ Long getCommRequestCountByCampaignTypeAndOrgaUnit(String campaignId,
     INNER JOIN su.campaign c
     INNER JOIN c.visibilities vi
     INNER JOIN su.interviewer i
-    INNER JOIN cr.communicationTemplate ct
+    INNER JOIN CommunicationTemplateDB ct ON ct.communicationTemplateDBId.campaignId = cr.communicationTemplateDBId.campaignId
```

Comment:
as CommunicationTemplate is declared in entity, 
INNER JOIN cr.communicationTemplate should be possible here